packages/react/src/search/IconCross.tsx

```diff
@@ -0,0 +1,13 @@
+export const IconCross = () => (
+  <svg
+    height="16"
```

Comment:
this component needs to accept `className` for decorator styling
```suggestion
export const IconSearch = ({ className }: { className?: string }) => (
  <svg
    className={className}
    height="16"
```